os/include/tinyara/input/ist415.h

```diff
@@ -0,0 +1,315 @@
+/****************************************************************************

```

Comment:
I think, this file can be in os/drives/input.

Reply:
done